nca/Parsers/IstioTrafficResourcesYamlParser.py

```diff
@@ -69,6 +70,7 @@ def parse_gateway(self, gateway_resource, gateway_file_name):
             peers &= self.peer_container.get_peers_with_label(key, [val])
         if not peers:
             self.warning(f'selector {selector} does not reference any pods in Gateway {gtw_name}. Ignoring the gateway')
+            self.missing_gw_peers = True
```

Comment:
I would be more specific to consider only gateway with selector `istio: ingressgateway`

Reply:
Done